TYPO3.Flow/Classes/TYPO3/Flow/Security/Cryptography/RsaWalletServicePhp.php

```diff
@@ -13,6 +13,7 @@
 
 use TYPO3\Flow\Annotations as Flow;
 use TYPO3\Flow\Security\Exception as SecurityException;
+use TYPO3\Flow\Security\Exception;
```

Comment:
Is there a reason why this is used twice?
I cannot say much about the crypto part.

Reply:
None, I simply didn't see this and auto-imported what I used.